src/main/java/edu/cornell/kfs/module/purap/businessobject/options/CollegeLevelOrganizationValuesFinder.java

```diff
@@ -34,4 +30,8 @@
         return keyValues;
     }
 
+    public void setiWantDocumentService(IWantDocumentService iWantDocumentService) {
```

Comment:
Can you please confirm that this setter is named correctly for being invoked by Spring, even though the first "i" is lowercased? Does it need to be renamed to "setIWantDocumentService" instead?

Reply:
I tested the I want document locally, and the college/division select list was populated.  I think that confirms this is working as expected.  Please let me know if you think something else should be checked.

Reply:
Yes, you're correct that this is the right naming for this setter, after I did some searching online and experimented with Eclipse's getter/setter generation feature. Sorry for the confusion.